diff -Nur -x '*.orig' -x '*~' nautilus-python-0.5.0/examples/README.in nautilus-python-0.5.0.new/examples/README.in
--- nautilus-python-0.5.0/examples/README.in	2008-01-11 12:07:15.000000000 +0100
+++ nautilus-python-0.5.0.new/examples/README.in	2008-04-11 19:30:44.000000000 +0200
@@ -1,6 +1,6 @@
 To try any of the examples, copy them over to:
 
-    @NAUTILUS_LIBDIR@/nautilus/extensions-1.0/python/
+    @NAUTILUS_LIBDIR@/nautilus/extensions-2.0/python/
 or:
     ~/.nautilus/python-extensions/
 
diff -Nur -x '*.orig' -x '*~' nautilus-python-0.5.0/nautilus-python.pc.in nautilus-python-0.5.0.new/nautilus-python.pc.in
--- nautilus-python-0.5.0/nautilus-python.pc.in	2008-01-11 12:07:15.000000000 +0100
+++ nautilus-python-0.5.0.new/nautilus-python.pc.in	2008-04-11 19:30:54.000000000 +0200
@@ -4,4 +4,4 @@
 
 prefix=@prefix@
 libdir=${prefix}/lib
-pythondir=@NAUTILUS_LIBDIR@/nautilus/extensions-1.0/python
+pythondir=@NAUTILUS_LIBDIR@/nautilus/extensions-2.0/python
diff -Nur -x '*.orig' -x '*~' nautilus-python-0.5.0/nautilus-python.spec.in nautilus-python-0.5.0.new/nautilus-python.spec.in
--- nautilus-python-0.5.0/nautilus-python.spec.in	2008-01-11 12:07:15.000000000 +0100
+++ nautilus-python-0.5.0.new/nautilus-python.spec.in	2008-04-11 19:31:07.000000000 +0200
@@ -45,7 +45,7 @@
 %defattr(-,root,root,-)
 %doc AUTHORS ChangeLog NEWS
 %{_libdir}/nautilus-python
-%{_libdir}/nautilus/extensions-1.0
+%{_libdir}/nautilus/extensions-2.0
 %{_libdir}/pkgconfig/nautilus-python.pc
 
 %changelog
diff -Nur -x '*.orig' -x '*~' nautilus-python-0.5.0/README nautilus-python-0.5.0.new/README
--- nautilus-python-0.5.0/README	2008-01-11 12:07:15.000000000 +0100
+++ nautilus-python-0.5.0.new/README	2008-04-11 19:31:42.000000000 +0200
@@ -6,7 +6,7 @@
 
 For examples and documentation check the examples sub directory.
 
-Note that scripts are loaded from $prefix/lib/nautilus/extensions-1.0/python
+Note that scripts are loaded from $prefix/lib/nautilus/extensions-2.0/python
 
 Try to copy test.py to that directory for an example
 
diff -Nur -x '*.orig' -x '*~' nautilus-python-0.5.0/src/nautilus-python.c nautilus-python-0.5.0.new/src/nautilus-python.c
--- nautilus-python-0.5.0/src/nautilus-python.c	2008-04-11 19:28:51.000000000 +0200
+++ nautilus-python-0.5.0.new/src/nautilus-python.c	2008-04-11 19:31:29.000000000 +0200
@@ -332,7 +332,7 @@
 
 	all_types = g_array_new(FALSE, FALSE, sizeof(GType));
 
-	nautilus_python_load_dir(module, NAUTILUS_LIBDIR "/nautilus/extensions-1.0/python");
+	nautilus_python_load_dir(module, NAUTILUS_LIBDIR "/nautilus/extensions-2.0/python");
 	user_extensions_dir = g_strdup_printf("%s/.nautilus/python-extensions/",
 										  g_get_home_dir());
 	nautilus_python_load_dir(module, user_extensions_dir);
