diff -urN nautilus-python-0.5.1/src/nautilus-python.c nautilus-python-0.5.1.new/src/nautilus-python.c
--- nautilus-python-0.5.1/src/nautilus-python.c	2008-09-16 21:00:19.000000000 +0200
+++ nautilus-python-0.5.1.new/src/nautilus-python.c	2008-10-07 20:36:12.000000000 +0200
@@ -206,8 +206,8 @@
 	if (Py_IsInitialized())
 		return TRUE;
 
-  	debug("g_module_open " PY_LIB_LOC "/libpython" PYTHON_VERSION "." G_MODULE_SUFFIX);
-	libpython = g_module_open(PY_LIB_LOC "/libpython" PYTHON_VERSION "." G_MODULE_SUFFIX, 0);
+  	debug("g_module_open " PY_LIB_LOC "/libpython" PYTHON_VERSION "." G_MODULE_SUFFIX ".1.0");
+	libpython = g_module_open(PY_LIB_LOC "/libpython" PYTHON_VERSION "." G_MODULE_SUFFIX ".1.0", 0);
 	if (!libpython)
 		g_warning("g_module_open libpython failed: %s", g_module_error());
 	debug("Py_Initialize");
